include/zephyr/net/zperf.h

```diff
@@ -43,6 +43,8 @@ struct zperf_upload_params {
 
 struct zperf_download_params {
 	uint16_t port;
+	struct in6_addr sin6_addr;    /* IPv6 address */
+	struct in_addr sin_addr;      /* IPv4 address */
```

Comment:
These three could be replaced by a single `struct sockaddr addr` which can hold the port number and IP address.